### PR TITLE
fix: remove some deprecated calls

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

File: core/corehttp/logs.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func LogOption() ServeOption {
 			w.WriteHeader(200)
 			wnf, errs := newWriteErrNotifier(w)
 			lwriter.WriterGroup.AddWriter(wnf)
-			log.Event(n.Context(), "log API client connected") //nolint deprecated
+			log.Info("log API client connected")
```
Comment (Stebalien) [inline core/corehttp/logs.go:53]: Apparently we still need this...

> grep "log API client connected" log_out

fails.

I _thought_ we were sending events here now...

**Review by marten-seemann — APPROVED** [review]
Changes lgtm me, not sure what to make of the logging change though.

**Comment by BigLep** [comment]
@Stebalien : will you update the logging or adjust the test so this passes?

**Comment by Stebalien** [comment]
I've reversed the change for now.

**Comment by Stebalien** [comment]
CI is broken due to https://github.com/ipfs/go-ipfs-api/pull/248.

**Pull request merged**